<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>动力系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="tree_database">
          <column>
           <property name="text">
            <string>选择文件</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>反应堆</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>转速减小</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>热阱</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>甩负荷</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>稳态</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_param">
          <column>
           <property name="text">
            <string>选择参数</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>电功率</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>反应堆功率</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>气轮机功率</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>压缩机功率</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>反应堆入口温度</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>反应堆出口温度</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>气轮机出口温度</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>压缩机入口温度</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>压缩机出口温度</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1,3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>播放速率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="box_speed">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="text">
             <string>播放长度</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="box_length">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1" columnstretch="2,4">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>时间</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_data_name">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>参数名称</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" rowspan="2">
              <widget class="QLineEdit" name="text_data">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="1" column="0" rowspan="2">
              <widget class="QLineEdit" name="text_time">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="canvas"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_insert"/>
   <addaction name="action_plot"/>
   <addaction name="action_tongji"/>
   <addaction name="action_analysis"/>
   <addaction name="action_option"/>
   <addaction name="action_help"/>
  </widget>
  <action name="action_insert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>数据录入</string>
   </property>
  </action>
  <action name="action_plot">
   <property name="text">
    <string>数据可视化</string>
   </property>
  </action>
  <action name="action_tongji">
   <property name="text">
    <string>数据统计</string>
   </property>
  </action>
  <action name="action_analysis">
   <property name="text">
    <string>综合分析</string>
   </property>
  </action>
  <action name="action_option">
   <property name="text">
    <string>系统设置</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
